web/src/main/java/org/wso2/testgrid/web/plugins/AWSArtifactReader.java

```diff
@@ -34,37 +38,32 @@
  */
 public class AWSArtifactReader implements ArtifactReadable {
 
-    private static final String AWS_ACCESS_KEY = "AWS_ACCESS_KEY_ID";
-    private static final String AWS_SECRET_KEY = "AWS_SECRET_ACCESS_KEY";
     private final AmazonS3 amazonS3;
     private final String bucketName;
 
     /**
      * Creates an instance of {@link AWSArtifactReader} for the given region and bucket name.
      * <p>
-     * Please do note that the {@value AWS_ACCESS_KEY} and {@value AWS_SECRET_KEY} environment values should be set
+     * Please do note that accessKey and secretKey values should be set in a properties file
      * in order to authenticate to the AWS.
      *
      * @param region region where the S3 bucket is located
      * @param bucket name of the bucket
-     * @throws ArtifactReaderException thrown if the given parameters are null or empty or if
-     *                                 the {@value AWS_ACCESS_KEY} and {@value AWS_SECRET_KEY} environment variable
-     *                                 values are not set
+     * @throws ArtifactReaderException thrown if the given parameters are null or empty
      */
     public AWSArtifactReader(String region, String bucket) throws ArtifactReaderException {
-        String awsIdentity = System.getenv(AWS_ACCESS_KEY);
-        String awsSecret = System.getenv(AWS_SECRET_KEY);
-        if (StringUtil.isStringNullOrEmpty(awsIdentity) || StringUtil.isStringNullOrEmpty(awsSecret)) {
-            throw new ArtifactReaderException("AWS Credentials must be set as environment variables");
-        }
         if (StringUtil.isStringNullOrEmpty(region)) {
             throw new ArtifactReaderException("AWS S3 bucket region is null or empty");
         }
         if (StringUtil.isStringNullOrEmpty(bucket)) {
             throw new ArtifactReaderException("AWS S3 bucket name is null or empty");
         }
+        Path configFilePath = Paths.get(TESTGRID_HOME, "testgrid-web-config.properties");
```

Comment:
Please use the TestGridUtil to get the testgrid_home. This const does not exist anymore. :)
 
https://github.com/wso2/testgrid/pull/436/files#diff-6ba7f91913f30091dfafa23448132eb3L47

Reply:
Fixed in 7178862